clusterloader2/pkg/test/test.go

```diff
@@ -65,6 +62,14 @@ func RunTest(f *framework.Framework, clusterLoaderConfig *config.ClusterLoaderCo
 			}
 		}
 	}
+
 	ctx := CreateContext(clusterLoaderConfig, f, state.NewNamespacesState())
+	mapping := make(map[string]interface{})
```

Comment:
Maybe just directly do:

```
mapping := map[string]interface{"Nodes": clusterLoaderConfig.ClusterConfig.Nodes}
```

Reply:
Also minor nit: Move this line to after getting the test config step.

Reply:
>mapping := map[string]interface{"...

Done.

>Also minor nit: Move this line to after getting the test config step.

I've moved it one line down. Not sure if that's what you meant.